htcondor/templates/osdf_rosetta_distribution.txt

```diff
@@ -1,3 +1,3 @@
-osdf:///chtc/staging/sgelman2/squid_rosetta_2023-10-30_21-53-58/rosetta_min_enc.tar.gz.aa
-osdf:///chtc/staging/sgelman2/squid_rosetta_2023-10-30_21-53-58/rosetta_min_enc.tar.gz.ab
-osdf:///chtc/staging/sgelman2/squid_rosetta_2023-10-30_21-53-58/rosetta_min_enc.tar.gz.ac
\ No newline at end of file
+/home/bryce.johnson/metl-sim/notebooks/downloads/rosetta_min_enc_v2.tar.gz.aa
```

Comment:
The changes in this directory look like they could break other metl-sim workflows.

Reply:
It wouldn't, since the user must redefine the current path. Currently it is sam's direct path.